spec/models/orchestration_stack.rb

```diff
@@ -4,6 +4,6 @@ class OrchestrationStack < ActiveRecord::Base
   has_ancestry
 
   belongs_to :ext_management_system, :foreign_key => :ems_id
-  has_many   :resources,  :dependent => :destroy, :foreign_key => :stack_id, :class_name => "OrchestrationStackResource"
-  alias_method :orchestration_stack_resources,  :resources
+  has_many   :resources, :dependent => :destroy, :foreign_key => :stack_id, :class_name => "OrchestrationStackResource"
+  alias orchestration_stack_resources resources
```

Comment:
ditto on this fix.

OrchestrationStack has always been confusing for me and I thought there may be some subclasses with it